frontend/components/Settings/Settings.tsx

```diff
@@ -79,6 +84,9 @@ export const Settings: React.FC<{
         } else if (currPassword !== "" && newPasswordScore < 20) {
             setNewPasswordError("Please provide a secure enough password");
             error = true;
+        } else if (newPassword !== retypePassword) {
+            setNewPasswordError("The provided passwords must be the same");
```

Comment:
Use the same message as from the login page to keep it consistent.

Reply:
Maybe place the error message in a field under the retypePassword field, as in the login page.